attackables.h

```diff
@@ -0,0 +1,24 @@
+//
+// Created by MassiveAtoms on 2/5/2019.
+//
+
+#ifndef TRPNGINE_ATTACKACKABLES_H
+#define TRPNGINE_ATTACKACKABLES_H
+
+#endif //TRPNGINE_ATTACKACKABLES_H
+
+#include <string>
+#include <stdexcept>
+
+
+class Attackable {
+public:
+    std::string a_name;
+    int health;
+    int strength;
+    int armour;
+
+    Attackable(std::string name, int hp, int str, int arm);
```

Comment:
```suggestion
    Attackable(std::string,int,int,int);
    
    Double check this, but I don't think we need to specify a name here, only when we call the function ( in attackables.cpp)
```

Reply:
Yeah, I thought I removed that. Will do so